server/server.go

```diff
@@ -99,9 +100,17 @@ func isLambda() bool {
 
 func (cs *cribbageServer) serve() error {
 	router := gin.Default()
+	router.Use(cors.New(getCORSConfig()))
+
 	cs.addRESTRoutes(router)
 
 	if isLambda() {
+		router.GET(`/backdoor`, func(c *gin.Context) {
+			c.String(
+				http.StatusOK,
+				`The front door is open. Come on in!`,
+			)
+		})
```

Comment:
Do we need to keep this?

Reply:
:shrug: I like it. It's an easy way to test that the lambda is deployed.